rmc/utils/mpc.py

```diff
@@ -1040,7 +1040,10 @@ def liftover_mpc(
     freeze: int = CURRENT_FREEZE,
     gnomad_version: str = CURRENT_GNOMAD_VERSION,
     remove_failed_sites: bool = True,
+    dedup_mpc: bool = False,
     overwrite_temp: bool = False,
+    failed_sites_str: str = "locus_fail_liftover",
+    remove_liftover_annotations: Set[str] = {"new_locus", "new_alleles"},
```

Comment:
Just want to double check, there's also `original_locus`, `original_alleles`, and `ref_allele_mismatch`. We're leaving those in there? (Can't remember what we did for release)

Reply:
yep we left those but can remove
```
hailctl describe gs://regional_missense_constraint/MPC/2.1.1/7/mpc_liftover38_dedup.ht
----------------------------------------
File Type: Table
    Partitions: 9761
    Rows: 67996604
    Empty partitions: 0
    Min(rows/partition): 629
    Max(rows/partition): 165302
    Median(rows/partition): 6251
    Mean(rows/partition): 6966
    StdDev(rows/partition): 3899
----------------------------------------
Global fields:

----------------------------------------
Row fields:
    'locus': locus<grch38>
    'alleles': array<str>
    'mpc': float64
    'transcript_grch37': str
    'original_locus': locus<grch37>
    'original_alleles': array<str>
    'ref_allele_mismatch': bool
----------------------------------------
Key: ['locus', 'alleles']
----------------------------------------
```